test/test.ts

```diff
@@ -427,3 +427,70 @@ test('can open a specific channel twice', async t => {
   const ch2 = await conn.channel(2)
   t.is(ch2, ch)
 })
+
+
+test('set basic flow on channel', async t => {
+  const amqp = new AMQPClient("amqp://127.0.0.1")
+  const conn = await amqp.connect()
+  const ch = await conn.channel()
+  await t.notThrowsAsync(async () =>  await ch.basicFlow(true))
+})
+
+test('can resolve promise on channel', async t => {
+  const amqp = new AMQPClient("amqp://127.0.0.1")
+  const conn = await amqp.connect()
+  const ch = await conn.channel()
+  t.false(ch.resolvePromise())
+})
+
+test('confirming unknown deliveryTag', async t => {
+  const amqp = new AMQPClient("amqp://127.0.0.1")
+  const conn = await amqp.connect()
+  const ch = await conn.channel()
+  t.notThrows(() => ch.publishConfirmed(1, false, false))
```

Comment:
this doesn't make sense?? publishConfirm is an internal method. can't we excerise the publish confirm code with a realistic test?

Reply:
The happy path is already covered with a "realistic test". If it doesn't make sense to test what happens if we try to confirm a unknown deliverytag, what's the reason for having that code path?

Reply:
Ok, alright, but maybe we should do something more than just not throw. Do we log? Should we have an on('error') emitter?